crates/ruff_db/src/source.rs

```diff
@@ -133,7 +133,7 @@ struct SourceTextInner {
 #[derive(Eq, PartialEq)]
 enum SourceTextKind {
     Text(String),
-    Notebook(Notebook),
+    Notebook(Box<Notebook>),
```

Comment:
Notebooks are massive and clippy complaint that the enum variants where significantly different in size (24 vs > 500)